tests/integration/security/fuzz/fuzz_test.go

```diff
@@ -36,20 +36,95 @@ const (
 
 	dotdotpwn = "dotdotpwn"
 	wfuzz     = "wfuzz"
+	jwtTool   = "jwttool"
 
-	authzDenyPolicy = `
+	authzDenyPolicy1 = `
 apiVersion: security.istio.io/v1
 kind: AuthorizationPolicy
 metadata:
-  name: policy-deny
+  name: policy-deny-page
 spec:
   action: DENY
   rules:
   - to:
     - operation:
         paths: ["/private/secret.html"]
 `
-	jwtTool            = "jwttool"
+
+	authzDenyPolicy2 = `
+apiVersion: security.istio.io/v1
+kind: AuthorizationPolicy
+metadata:
+  name: policy-deny-page-with-wildcard
+spec:
+  action: DENY
+  rules:
+  - to:
+    - operation:
+        paths: ["/private/secre*.html"]
+`
+
+	authzDenyPolicy3 = `
+apiVersion: security.istio.io/v1
+kind: AuthorizationPolicy
+metadata:
+  name: policy-deny-page-with-wildcard
+spec:
+  action: DENY
+  rules:
+  - to:
+    - operation:
+        paths: ["/private/secret*.html"]
+`
```

Comment:
Is there a significant testing/coverage difference between these two deny policies?